### ISSUE TITLE
is it possible to stream audio from windows to windows?

### ISSUE THREAD

**Issue by comsyspro** [pgaskin/pulseaudio-win32#14]
i want to stream sound from a windows (pulseaudio server) to a windows (pulseaudio client) but i can't hear sound from windows (pulseaudio server)

following steps were done:
- running pulseaudio on windows (server ip 192.168.0.11) and windows (client ip 192.168.0.22)
- on windows client executing `pactl.exe load-module module-tunnel-source-new server=192.168.0.11`

result:
- the windows server shows a connection of the windows client, but no sound can be heared on the windows client.
```
: [(null)] ../../src/pulseaudio/src/pulsecore/client.c: Created 0 "Native client (TCP/IP client from 192.168.0.22:51060)"
: [(null)] ../../src/pulseaudio/src/pulsecore/protocol-native.c: Client authenticated anonymously.
```

is it possible that the windows server have to redirect somehow the audio signal to the soundcard driver? is this possible at all to get a remote stream from windows to windows? any advice?

**Comment by pgaskin** [comment]
Not currently; this would require a better pulseaudio module for interacting with WASAPI rather than waveout.

I don't currently have a personal need for this, nor do I have much spare time (hence the lack of updates recently), but if I ever need it again, I'll probably work on writing a SPA plugin for pipewire and porting it to windows instead, as that API will be much easier to integrate, and pipewire is the successor to pulseaudio.

You might be interested in [Scream](https://github.com/duncanthrax/scream) as an alternative method for streaming sound between machines.

**Comment by comsyspro** [comment]
Yes I already tried Scream and it works also. Do you know this software https://vb-audio.com/Cable/ and would it be possible to use this for pulsaudio on windows somehow to create audio working form win to win with the actual code of pulseaudio?
